apps/aot/src/components/navigation/index.tsx

```diff
@@ -35,27 +35,30 @@ export async function Navigation() {
           <NavLinksWithYear />
           <NavLink href="/about">About</NavLink>
           <Link
-            className="donate-btn relative w-fit overflow-hidden rounded-full px-4 py-2 text-black/50 duration-300 hover:bg-[#eed15f] md:hidden dark:text-white/80 dark:hover:bg-[#bea74b44]"
+            className="donate-btn relative w-fit overflow-hidden rounded-full px-4 py-2 text-black/60 duration-300 hover:bg-[#eed15f] md:hidden dark:text-white/80 dark:hover:bg-[#bea74b44]"
             href="/support"
           >
             Support Us
           </Link>
           <Separator className="md:hidden dark:bg-white/20" />
           <NavLink
             href={DISCORD_INVITE_LINK}
-            className="flex flex-row items-center gap-2 text-black/40 md:hidden dark:text-white/60"
+            className="flex flex-row items-center gap-2 text-black/40 dark:text-white/60"
+            linkClassName="md:hidden"
```

Comment:
`NavLink` passes `className` to a child rendered inside a `<Link>`. That means the `<Link>` is not hidden when the child's `md:hidden` is in effect. A `<Link>` with no children still gets <kbd>Tab</kbd> focus, which is an accessibility no-no.

Aside: this is one of the reasons why having a `className` prop apply to a _child_ component, rather than the root of what's rendered, is confusing. I recommend against it. 😬

Reply:
yeah thats odd. will followup and clean that up later